crates/cheatnet/src/runtime_extensions/call_to_blockifier_runtime_extension/panic_data.rs

```diff
@@ -32,16 +32,16 @@ mod test {
     fn string_extracting_panic_data() {
         let cases: [(&str, Option<Vec<Felt252>>); 4] = [
             (
-                "Beginning of trace\nGot an exception while executing a hint: Hint Error: Execution failed. Failure reason: \"PANIK, DAYTA\".\n
+                "Beginning of trace\nGot an exception while executing a hint: Custom Hint Error: Execution failed. Failure reason: 0x434d3232 ('PANIK, DAYTA').\n
                  End of trace",
                 Some(vec![Felt252::from(344_693_033_291_u64), Felt252::from(293_154_149_441_u64)])
             ),
             (
-                "Got an exception while executing a hint: Hint Error: Execution failed. Failure reason: \"AYY, LMAO\".",
+                "Got an exception while executing a hint: Custom Hint Error: Execution failed. Failure reason: 0x434d3232 ('AYY, LMAO').",
                 Some(vec![Felt252::from(4_282_713_u64), Felt252::from(1_280_131_407_u64)])
             ),
             (
-                "Got an exception while executing a hint: Hint Error: Execution failed. Failure reason: \"\".",
+                "Got an exception while executing a hint: Custom Hint Error: Execution failed. Failure reason: 0x0 ('').",
```

Comment:
cc @Arcticae 
please verify these changes are okay

Reply:
If panic parsing tests pass it will be correct